tests/Functional/App/config/packages/cache.yaml

```diff
@@ -0,0 +1,19 @@
+framework:
+    cache:
+        # Unique name of your app: used to compute stable namespaces for cache keys.
```

Comment:
Some comments here could be removed

Reply:
👍  I simply created a new Symfony project and did not change anything in there.